rotkehlchen/api/v1/schemas.py

```diff
@@ -2147,10 +2147,6 @@ class BaseAddressbookSchema(Schema):
     book_type = SerializableEnumField(enum_class=AddressbookType, required=True)
 
 
-class AddressbookAddressesSchema(BaseAddressbookSchema):
```

Comment:
This is a mistake. This was previous extending `BaseAddressbookSchema` and you are replacing this directly with `OptionalAddressesListSchema` which is a mistake.

You are removing the `book_type` argument and adding an unneeded `async_query` argument.

What you need to do is keep this but have an `OptionalAddressesListSchema` **without** the `AsyncQueryArgumentSchema`. And extend this here too.

And later down in `DetectTokensSchema` also add `AsyncQueryArgumentSchema`.

Reply:
Oh yeah yeah, I understand it. Just was a bit inattentive in the evening after spending my brain resource onto understanding why the test (with cache) was failing.